exercise_01.py

```diff
@@ -0,0 +1,19 @@
+def pyramid_of_numbers(n):
```

Comment:
@DanielVelinrb , great! I don't see more changes to suggest. However, your function is not aligning properly the numbers when `n >= 10`. In addition, you are adding an extra blank line at the beginning that is annoying.